tw-entrypoints-starter/src/main/java/com/transferwise/common/entrypoints/EntryPointsAutoConfiguration.java

```diff
@@ -67,13 +70,20 @@ public static TableAccessStatisticsBeanPostProcessor twEntryPointsTableAccessSta
     return new TableAccessStatisticsBeanPostProcessor(beanFactory);
   }
 
+  @Bean
+  @ConditionalOnProperty(name = "tw-entrypoints.tas.enabled", havingValue = "true", matchIfMissing = true)
+  public static MeterFilter twEntryPointsTableAccessStatisticsMeterFilter() {
```

Comment:
Do the beans need to use static methods?

Reply:
nope youre right. fixed that.